recipes/iacs-ipac-reader/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "iacs-ipac-reader" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/iacs_ipac_reader-{{ version }}.tar.gz
+  sha256: 7464bd5a3f9ad66a7e2dfe6850aa47bfa79b1f75c90b2dcc0dca0fcb1c199924
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - h5py >=3.5.0
+    - napari ==0.4.12
+    - napari-plugin-engine ==0.2.0
```

Can we relax this a little bit?

```suggestion
    - napari-plugin-engine >=0.2.0
```